openstack/barbican/alerts/openstack-barbican.alerts

```diff
@@ -39,7 +39,7 @@ groups:
       severity: info
       tier: os
       service: 'barbican'
-      context: 'barbican'
+      context: 'barbican-cert-expiry'
```

Comment:
This is a bit redundant as the service is already `barbican`. 
Preferably something broader or use `certificate` if you expect more alerts here, in which case automating the renewal might be useful.